deploy/2-configure-feeds.ts

```diff
@@ -120,6 +135,108 @@ const assets: Assets = {
       oracle: "pyth",
       price: "159990000000000000000",
     },
+    {
+      token: "TRX",
+      address: "0x7D21841DC10BA1C5797951EFc62fADBBDD06704B",
+      oracle: "chainlink",
+      price: "159990000000000000000",
+    },
+    {
+      token: "TRX",
```

Comment:
This is the old TRX token. I assume under the hood, by using the same token symbol, we'll use the same chainlink feed, that seems correct. Binance didn't rename this token in testnet. I would add just a comment to identify this special case at least in this script

```suggestion
      token: "TRX", // OLD TRX
```